src/components/PaperUIKit/Button/Button.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Button as ButtonUIPaper } from 'react-native-paper';
+import {  blueLight, blueMedium } from "../../../constants/colors";
+import styles from "./styles";
+
+export enum ButtonType {
+  OUTLINED = 'outlined',
+  TEXT = 'text',
+  CONTAINED = 'contained',
+}
+
+interface ButtonProps {
+  type: ButtonType,
+  title: string,
+  containerStyle?: object,
+  onPress: () => void
+}
+
+const getLabelStyleByType = (type: ButtonType): object => {
```

Comment:
It's better to place all 'stylish' functions in 'styles.js'

so u will import  from there al style stuff:

`import { styles, getLabelStyleByType, getColorByType, getContainerStyleByType } from "./styles";`

Reply:
fixed